app/views/dashboard/saml_login.html.haml

```diff
@@ -0,0 +1,15 @@
+<!--[if IE 9]><html class="ie9 login-pf"><![endif]-->
+%html.login-pf{:class => get_vmdb_config[:server][:custom_login_logo] ? '' : 'rcue-login', :lang => I18n.locale.to_s.sub('-', '_')}
+  %head
+    = favicon_link_tag
+    = stylesheet_link_tag 'application'
+    = javascript_include_tag 'application'
+
+  %body
+    - if api_auth_token && validation_url
+      :javascript
+        sessionStorage.miq_token = '#{j_str api_auth_token}';
+        window.location.href = '#{j_str validation_url}';
+    - else
+      :javascript
+        window.location.href = '/dashboard/logout';
```

Comment:
the window.location = ... worked just as well.  Updated and repushed PR.  Thanks !!